src/hooks/posters.ts

```diff
@@ -71,24 +72,55 @@ export const usePosters = (posterHallId: string) => {
   const fuseVenues = useMemo(
     () =>
       new Fuse(filteredPosterVenues, {
```

Comment:
```suggestion
      // Ref: https://fusejs.io/api/options.html
      new Fuse(filteredPosterVenues, {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1460/commits/9d293acfb44e50f362e410eec6042786a34aa6fd